pkg/platform/runtime/runtime.go

```diff
@@ -68,7 +68,7 @@ func newRuntime(target setup.Targeter, an analytics.Dispatcher, svcModel *model.
 
 	err := rt.validateCache()
 	if err != nil {
-		return nil, err // do not wrap; could be NeedsUpdateError, NeedsStageError, etc.
+		return rt, err
```

Comment:
Is my comment about not wrapping invalid? If so, that's fine. Otherwise, please leave it, as it's easy to change this into a wrap later during a refactor.

Reply:
The errs.Matches and other unwrapping error helpers should be able to inspect throughout the chain of errors, so wrapping shouldn't need to be avoided.